fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -1595,30 +1596,46 @@ public void gsonPreProcess() throws IOException {
     @Override
     public void gsonPostProcess() throws IOException {
         super.gsonPostProcess();
+        // only range partition need to recover partitionRefTableExprs
+        if (!(partitionInfo instanceof ExpressionRangePartitionInfo)) {
```

Comment:
Does there exist EditLog imcompatibility issue?

Reply:
Don't worry about this. only `ExpressionRangePartitionInfo` is supported by mv for now.